modules/home/assets/js/components/home-screen.js

```diff
@@ -7,14 +7,16 @@ const HomeScreen = ( props ) => {
 		embedUrl = 'https://www.youtube.com/embed/icTcREd1tAg?si=40E8D1hdnu26-TXM';
 
 	return (
-		<Container sx={ { display: 'flex', flexDirection: 'column', gap: { xs: 1, md: 3 }, py: { xs: 2, md: 6 }, maxWidth: { md: '990px' } } }>
-			<TopSection
-				videoUrl={ videoUrl }
-				embedUrl={ embedUrl }
-				topData={ props.homeScreenData.data.top[ 0 ] }
-				createNewPageUrl={ props.homeScreenData.createNewPageUrl }
-			/>
-		</Container>
+		<Box maxWidth="xl" sx={ { pr: 1 } }>
```

Comment:
I am not sure if I understand the reason why we need this `<Box>`.

Reply:
it is a workaround to allow Container to have 1200 without x padding, while having some padding around it on smaller screens and to compensate for the padding left that comes from wp-dashboard.

Reply:
@elchugreeva It sounds okay. Can we huddle about it in the morning? I would like to understand better how it works.